installer/scripts/omsadmin.sh

```diff
@@ -346,6 +362,28 @@ set_proxy_setting()
     fi
 }
 
+set_omsagent_proc_limit()
```

Comment:
can we apply at group level or do we need to include nxautomation user too as it is also a user created by OMS agent